### PR TITLE
[IMP] pos: Improve UI and access controls across features

### PR DESCRIPTION
**Pull request by nesma-neha**
- Implemented employee login and update form placeholders for basic and advanced rights in the edit dashboard.
- Enforced primary payment method after order placement in pos_restaurant.
- Restricted basic user rights from creating products.
- Swapped positions of Lock and Print buttons, added a lock icon for clarity.
- Adjusted floor plan corner selector size.
- Hid zero dollar price for combos in the cart. 
 
Task ID: 3916835

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_hr/static/src/overrides/components/navbar/navbar.js [hunk 1]
```diff
@@ -22,4 +22,21 @@ patch(Navbar.prototype, {
...
+    get showEditPlanButton() {
+        if (
+            this.pos.config.module_pos_restaurant &&
+            (!this.pos.config.module_pos_hr || this.employeeIsAdmin())
+        ) {
+            return super.showEditPlanButton;
+        } else {
+            return false;
+        }
+    },
```
Comment (vlst-odoo) [inline addons/pos_hr/static/src/overrides/components/navbar/navbar.js:41]: this should be in the `pos_hr_restaurant` module

**Review by vlst-odoo — CHANGES_REQUESTED** [review]

**Comment by caburj** [comment]
fw-bot ignore

PR originated from master: https://github.com/odoo/odoo/pull/165416

**Comment by caburj** [comment]
robodoo ignore

**Comment by caburj** [comment]
robodoo fw=ignore

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**